src/Illuminate/Support/Fluent.php

```diff
@@ -24,9 +24,7 @@ class Fluent implements ArrayAccess, Arrayable, Jsonable, JsonSerializable
      */
     public function __construct($attributes = [])
     {
-        foreach ($attributes as $key => $value) {
-            $this->attributes[$key] = $value;
-        }
+        $this->attributes = $attributes;
```

Comment:
It can break code, if someone has extended this class and defined the 'attribute' property within the class.

Reply:
Also, what happens when `$attributes` is an object?

Reply:
@TBlindaruk @JayBizzle Thank you for clarifying this!